app/models/induction_coordinator_profiles_school.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class InductionCoordinatorProfilesSchool < ApplicationRecord
+  has_paper_trail
```

Comment:
I wonder if there is some way to tell Rails that we want to have paper trail with all the default models it creates? 

Oh, I am already scared to day it, is this something we want in our `ApplicationRecord`? Or is that too monkey-patchy for our taste?

Reply:
Would we always want to have it enabled on every model?  I could envisage some kind of base class or concern that was `Auditable` or something like that

Reply:
Ehh, always is such a strong word 😂 

On E&L certainly not. But this is not E&L land. Thoughts @ethanmills? @jamesrose?

I think `has_paper_trail` is that concern that you would end up devising. Base class might be more like it, cause then we avoid having `has_paper_trail` all over the place. 

And I guess if we went all the way and changed application record we would not need to add our own models for the default many-to-many relations. But it's also changing the global behaviour of application record, so it sounds a bit scary.

Reply:
Eh, I think this is fine. There are lots of things we don't particularly care about versioning